config/data/markdown/en/about.md

```diff
@@ -53,6 +53,15 @@ I currently work at [anders und sehr GmbH](https://andersundsehr.de) as my main
 I do not work fulltime as a freelancer, and I do not plan to change that in the foreseeable future.
 
 ~Timeline~
+---
```

Comment:
```suggestion
```

Reply:
To keep it consistent the same change should be done on the german about.md.
So you're sure to want it like this?

Reply:
Yup, it's necessary to prevent an empty timeline slot